wpeverest/everest-forms
# Notification option set to individual connections

### All Submissions:






### Changes proposed in this Pull Request:



This PR allows user to set notification ON/OFF for every emails connections individually.

### How to test the changes in this Pull Request:

1. Create Form.
2. GOTO Email Setting
3. Add Email Connections.
4. Set Notification ON/OFF
5. Verify the changes.

### Types of changes:




### Other information:




### Changelog entry

> Notification option set to individual connections.
